### ISSUE TITLE
Editor tab "PDF preview" button (enchancement)

### ISSUE THREAD

**Issue by bartosz-antosik** [James-Yu/LaTeX-Workshop#229]
This is clearly a feature request.

Would you consider reverting a "PDF Preview" button to editor's tab action area?

It has been there and I have created an icon following my complaint that the original one was blurry (#28).

I would imagine there should only be one icon and the action could be configurable to either display "PDF Preview in a Tab" or "PDF Preview in a Web Browser".

This tab for me (aside base functionality) allowed to quickly realize that "I am on a LaTeX document tab". ;-)

**Comment by James-Yu** [comment]
It would be just fine to backport the feature. However now we have an extra concern on vscode theme. Do we need to prepare a set of icons with different palette to adjust to light/dark themes?

**Comment by James-Yu** [comment]
I recalled that the code snippet was borrowed from Arduino extension. Will check their codebase first.

**Comment by James-Yu** [comment]
I'd use this as a temporary solution, since it looks good on both dark and light themes, and is not very blurry. Discussions and suggestions welcomed!

![image](https://user-images.githubusercontent.com/4210342/28992123-0425cc94-79c7-11e7-8ea9-30c42a14dd8e.png)

**Comment by bartosz-antosik** [comment]
Thank you for very prompt response to the proposition!

1. Attached are my icons with corrected colors for light/dark themes. The colors are exactly the same as VSCode's colors checked across few themes, so more or less should work. 

2. Here is how you connect light/dark icons in `package.json`:

```
    "contributes": {
        "menus": {
            "editor/title": [
                {
                    "when": "resourceLangId == latex",
                    "command": "latex-workshop.tab",
                    "group": "navigation",
                    "icon": {
                        "light": "media\\view-pdf-light.svg",
                        "dark": "media\\view-pdf-dark.svg"
                    }
                }
            ]
        }
    }, (...)
```

3. What do you think about configuring whether it takes to tab or web browser? (I think I use tab more often but still think would be nice to have users to decide)

[icons.zip](https://github.com/James-Yu/LaTeX-Workshop/files/1202241/icons.zip)

**Comment by James-Yu** [comment]
Yes I'd say the last point makes much sense. Let me implement it.

**Comment by James-Yu** [comment]
I have been thinking about how to implement the third enhancement without adding a new command. Unfortunately it seems to be impossible. The button itself must be a vscode-registered command, which must show in the quick menu, which can be really messy (have three `view PDF in xxx` commands). I'll close this issue for now but discussions and suggestions are always welcomed.

**Comment by bartosz-antosik** [comment]
Until few days back (when after your comment I did some tests) I was pretty sure that one can call proxy command which is registered (via `vscode.commands.registerCommand`) but not revealed in `package.json`. But no, One cannot. This would truly require revealing the command.

Maybe a command like `View PDF in Default Viewer` would not be much of a problem in your extension, but if your taste says it would then I absolutely agree.

P.S. I have tweaked the icons a bit to be more "in line" with VSCode icons, if you would likt them then voila:

[icons.zip](https://github.com/James-Yu/LaTeX-Workshop/files/1216340/icons.zip)
